Exercises/3-hello.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-const hello = null;
+const hello = name => console.log(`Hello, ${name}!`);
```

Comment:
```suggestion
const hello = name => {
  console.log(`Hello, ${name}!`);
};
```
We use single line functions when outer function need to return value of internal call.